aws/table_aws_macie2_finding.go

```diff
@@ -0,0 +1,316 @@
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "id",
+				Description: "The unique identifier for the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "finding_arn",
+				Description: "The Amazon Resource Name (ARN) of the finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("FindingArn"),
+			},
+			{
+				Name:        "finding_type",
+				Description: "The type of finding.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Type"),
+			},
+			{
+				Name:        "severity",
+				Description: "The severity level of the finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "status",
+				Description: "The status of the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "created_at",
+				Description: "The date and time, in UTC and extended ISO 8601 format, when the finding was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "updated_at",
+				Description: "The date and time, in UTC and extended ISO 8601 format, when the finding was last updated.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "description",
+				Description: "A description of the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "category",
+				Description: "The category of the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "count",
+				Description: "The total number of occurrences of the finding.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "resources_affected",
+				Description: "The resources that the finding applies to.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "sample",
+				Description: "A sample of the data that triggered the finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "classification_details",
+				Description: "The details of the classification that produced the finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "remediation",
+				Description: "Information about the remediation steps for the finding.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "archived",
+				Description: "Specifies whether the finding is archived.",
+				Type:        proto.ColumnType_BOOL,
+			},
+			{
+				Name:        "source_region",
+				Description: "The AWS Region where the finding was generated.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "source_account_id",
+				Description: "The AWS account ID for the account that owns the finding.",
+				Type:        proto.ColumnType_STRING,
+			},
+
+			// Steampipe standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
```

Comment:
Shouldn’t we add a transform function here? Does the API actually return a field named `title`?

Reply:
Yes, the API response has field named `Title`.